docs/learn/test-runner/writing-plugins.md

```diff
@@ -6,9 +6,301 @@ eleventyNavigation:
   order: 70
 ---
 
-...
+Not every use case will be covered by existing plugins 😱
+Therefore if you encounter a situation that requires some custom adjustments you can create a plugin yourself.
```

Comment:
Paragraphs should be separated be a newline, or the text should be continuous. Introducing a manual line break will break up the text uneven on different breakpoints.

That goes for this, but als other paragraphs below.

Reply:
not as far as I know? 🤔

single newlines are ignored and it's the same as to keep writing in a single line but you get the benefit that diffs are nicer (as each sentense has a separate line)

if you need a line break then you need to add a `br`

Reply:
Aah, you're right. Sorry!